### PR TITLE
Clean up batchedit remove and faceting

### PR DESCRIPTION
**Pull request by katdivyareddy10**
Screen Shot 2020-08-18 at 12 13 21 PM
Screen Shot 2020-08-18 at 12 13 31 PM

### CODE REVIEW

File: assets/js/components/BatchEdit/Remove.js [hunk 1]
```diff
@@ -27,15 +27,15 @@ export default function BatchEditRemove({
...
-        <div className="content">
-          <ul>
+        <div className="content has-background-danger-light">
+          <ul className="py-4">
             {removeItems.map((item) => {
               const { label, role, term } = splitFacetKey(item);
 
               return (
-                <li key={item} className="has-text-danger">
-                  {`${label} - ${term} - ${role}`}
```
Comment (adamjarling) [inline assets/js/components/BatchEdit/Remove.js:37]: @katdivyareddy10 @kdid  Do we want to keep `role` here for Contributor and Subject s?

File: assets/js/services/reactive-search.js [hunk 2]
```diff
@@ -28,8 +28,8 @@ export const REACTIVE_SEARCH_THEME = {
 // Documention: https://docs.appbase.io/docs/reactivesearch/v3/theming/classnameinjection/
 const facetClasses = {
   checkbox: "facet-checkbox",
-  label: "facet-item-label",
-  title: "facet-title",
+  label: "facet-item-label mx-2",
+  title: "facet-title mt-4",
```
Comment (adamjarling) [inline assets/js/services/reactive-search.js:32]: Clever!   Nice one.

File: assets/js/components/BatchEdit/Remove.js (hunk 1 above)
Comment (katdivyareddy10) [reply assets/js/components/BatchEdit/Remove.js:37]: I can remove it if it's not useful.

Comment (kdid) [reply assets/js/components/BatchEdit/Remove.js:37]: Keep the roles for contributor and subject. (only display the role label (not role id)) but removing:
Contributor 1 Term - contributor1id - (Author) 
Contributor 1- contributor1id -  (Publisher)

are two different things - users need to see the difference.

Comment (katdivyareddy10) [reply assets/js/components/BatchEdit/Remove.js:37]: Ok, I guess that means we're keeping role.

Comment (katdivyareddy10) [reply assets/js/components/BatchEdit/Remove.js:37]: If there are no other changes, could either of you review this PR?

File: assets/js/components/BatchEdit/Remove.js [hunk 3]
```diff
@@ -27,15 +27,15 @@ export default function BatchEditRemove({
...
-                </li>
+                <li key={item}>{`${label} - ${term} ${
+                  role ? "- " : ""
+                } ${role}`}</li>
```
Comment (adamjarling) [inline assets/js/components/BatchEdit/Remove.js:38]: @katdivyareddy10 It's small, but can we tighten up this ternary?  To look something like:
```
<li key={item}>{`${label} - ${term}`} {role ? `- ${role}` : ""}</li>
```

**Pull request merged**